src/server/ability.ts

```diff
@@ -14,13 +15,28 @@ function defineAbilityFor(user: any) {
   // AbilityBuilder https://casl.js.org/v5/en/guide/define-rules#ability-builder-class
   const {can, build} = new AbilityBuilder<AppAbility>(Ability)
 
-  if (user.roles?.includes('admin')) {
+  if (hasRoles(user, 'admin')) {
```

Comment:
`_.includes([1, 2, 3], 1, 2);` lo dash includes works like you’d expect and is probably the way to go IMO

Reply:
I forgot that lodash has an `includes`. It's not quite what I want though. That third argument is to specify an index.
![CleanShot 2022-03-10 at 18 07 36@2x](https://user-images.githubusercontent.com/694063/157782425-05034c43-7ae7-4b81-896d-51a4ce8a50ce.png)

I wanted to improve on the readability as well by not having to do:

```javascript
if (includes(user?.roles, 'publisher') || includes(user?.roles, 'editor')) {
  ...
}
```

The multi-role checks get kinda cumbersome to read imo.